### DISCUSSION TITLE
Co pilot no longer showing on VCS status bar

### DISCUSSION THREAD

**Discussion by AJBeere** [community/community#34621] [Copilot Conversations]
I opened VSC this morning and copilot is no longer available on the status bar. I checked my subscription, I still have the free trial for 53 days and have already provided payment details to subscribe when the free trial ends. 

When I look at my extensions, copilot is greyed out. Any suggestions on what I can do?

**Comment by davecheney** [comment]
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.
